sdk/src/main/java/io/radar/sdk/RadarJobScheduler.kt

```diff
@@ -64,7 +83,19 @@ class RadarJobScheduler : JobService() {
             this.time = time
         }
 
-        val sourceStr = extras.getString(EXTRA_SOURCE) ?: return false
+        val sourceStr = extras.getString(EXTRA_SOURCE)
+        Radar.logger.d(
+            "Start Location Job | " +
+                    "location = $location; " +
+                    "source = ${sourceStr}; " +
+                    "standbyBucket = ${Radar.batteryManager.getAppStandbyBucket()}; " +
```

Comment:
Something that could be interesting at some point is for `RadarLogger` to grab this state itself and add it to the log.  But I like the idea of starting small and only putting it in the most important paths right now (`handleLocation` and `startJob`).  Feels like it could also be really useful in the `RadarLocationReciever` log.

Reply:
Very nitpicky, but noting that `Start Location Job` is formatted differently than every other log line in the SDK (generally sentence case and `-ing` verbs like `Stopping foreground service` rather than `Stop Foreground Service`: https://github.com/radarlabs/radar-sdk-android/blob/master/sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt#L610)

If we want this to be readable and maybe customer-facing eventually, should we try to be consistent?

Reply:
If this were closed source or purely internal logging, wouldn't matter. But we should hold ourselves to a higher standard of consistency here. Same with release notes, code comments in open-source libs, etc. cc @tjulien @jsani-radar @jrtibbetts

Reply:
Agreed. We should create some common standards and enforce them through static analysis and code review.

Reply:
Added -ing's

Reply:
@nickpatrick @phil-brown good call, Added a project on the Q1 roadmap to create a logging standard and linter